src/components/helpers/choices.js

```diff
@@ -249,12 +249,18 @@ export default createReactClass({
       const rect = node.getBoundingClientRect();
       const top = rect.top;
       const windowHeight = window.innerHeight;
-      const height = windowHeight - top;
-      if (height !== this.state.maxHeight) {
+      const propsMaxHeight =
```

Comment:
@efhjones Doesn't sound too bad. I'm guessing that we will want the same max height for all choices dropdowns in the editor? If so it might be easier to pass `choicesMaxHeight` via the formatic config:

```js
  const configMaxHeight = this.props.config.getChoicesMaxHeight;
  //...
```

In the [default formatic config](https://github.com/zapier/formatic/blob/master/src/default-config.js) you could provide the default implementation of `getPropsMaxHeight` that would return `1`:

```js
  getChoicesMaxHeight {
    return 1;
  }
```

Then in our `formaticFieldsConfig` we could specify `0.75`. That way the editor will use that value, but nothing else changes.

Minor thing: I'm not sure if `choicesMaxHeight` is the best name here. That makes me think it's the actual max height, like in pixels or something. But actually it's the ratio of available height to actual max height. So... (drawing a blank here...) .. uhm `choicesMaxHeightRatio` or something? Not totally clear, but at least it doesn't lull me into thinking it's the actual max height.

Reply:
> If so it might be easier to pass choicesMaxHeight via the formatic config

That's an excellent point! Thanks! 

> That makes me think it's the actual max height, like in pixels or something

Great point as well! I'm struggling to come up with something better than `choicesMaxHeightRatio` so we'll go with that! 😄